.github/PULL_REQUEST_TEMPLATE

```diff
@@ -7,7 +7,7 @@ Thanks for sending a pull request!  Here are some tips for you:
   5. Please write your PR title to summarize what this PR proposes.
   6. If possible, provide a concise example to reproduce the issue for a faster review.
   7. If you want to add a new configuration, please read the guideline first for naming configurations in
-     'core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala'.
+     https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala
```

Comment:
Why do we need this change?

Reply:
This is just a minor improvement to make the contribution experience better, we send contributors to read something but they have to go and look it manually instead of easily accessing the link. Since it is not related to Avro upgrade I can remove it if you prefer.

Reply:
You can open a new pr for this change.

Reply:
+1 for @wangyum 's advice.
Please revert this change from this PR.

Reply:
+1 for @wangyum 's advice.

Reply:
ok will do